gatsby-node.js

```diff
@@ -115,13 +117,130 @@ exports.sourceNodes = async ({ actions }) => {
   return
 }
 
-// exports.onCreateNode = ({ actions, node }) => {
-//   const { type } = node.internal
-//   if (type !== `Member`) return
-//   const { bio } = node
-//   const bioAsHtml = markdownProcessor.process(bio)
-//   node.bio = bioAsHtml
-// }
+exports.onCreateNode = ({ node, actions }) => {
+  const { type } = node.internal
+  if (type === 'GhostAuthor') {
+    if (node.slug !== 'data-schema-author') {
+      actions.createNodeField({
```

Comment:
To be consistent w/ other functions either change all the other ones to use `actions.doSomething` or change this to be

```javascript
const { doSomething } = actions
...
doSomething()
```